README.md

```diff
@@ -57,19 +69,31 @@ Data allocation and manipulation.
 ## Constants
 
 <dl>
-<dt><a href="#recipes">recipes</a> : <code>Object</code></dt>
+<dt><a href="#services">services</a></dt>
+<dd><p>List helpful functions when dealing with collections.</p>
+</dd>
+<dt><a href="#recipes">recipes</a></dt>
 <dd><p>List of class declarations that can be used to specify attributes for a style of object / class.</p>
 </dd>
 </dl>
 
-## Typedefs
+## Functions
 
 <dl>
-<dt><a href="#completeResponse">completeResponse</a> : <code>Object</code></dt>
-<dd><p>Return results of the task.</p>
+<dt><a href="#parseTreeNext">parseTreeNext(treeNode)</a> ⇒ <code>IsTreeNode</code> | <code>null</code></dt>
+<dd><p>Figure this out</p>
+<ol>
+<li>Start at root (get root parent)</li>
+<li>Get first child (repeat until no children)</li>
+<li>Check next child</li>
+<li>Repeat 2</li>
+<li>Repeat 3</li>
+<li>If no next child, return to parent and repeat 3</li>
+<li>Stop at root (next is null and parent is null</li>
+</ol>
```

Comment:
These notes could be helpful. Since they appear in the readme it might be meaningful if they are written in a better format.